packages/plugin-googledrive-non-gms/src/main/java/com/openmobilehub/android/storage/plugin/googledrive/nongms/data/repository/NonGmsFileRepository.kt

```diff
@@ -236,69 +280,97 @@ internal class NonGmsFileRepository(
         return null
     }
 
-    suspend fun downloadFile(fileId: String): ByteArrayOutputStream {
-        val response = retrofitImpl
-            .getGoogleStorageApiService()
-            .downloadMediaFile(fileId = fileId, alt = MEDIA)
+    @VisibleForTesting
+    suspend fun simplyUploadFile(
```

Comment:
What do you think about renaming - I feel like word `simply` does not really explain when/for witch files it should be used. Small files upload is also the terminology [that OneDrive is using](https://learn.microsoft.com/en-us/onedrive/developer/rest-api/api/driveitem_put_content).
```suggestion
    suspend fun smallFileUpload(
```

Reply:
I hate to be picky, but `smallUploadFile` does not seem to be the correct English. I assume you want to keep it similar to the `uploadFile` method, so maybe `uploadSmallFile`?